packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -121,6 +121,10 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
       treeType: FlutterTreeType.widget,
     );
 
+    if (!inspectorController.firstLoadCompleted) {
+      ga.timeStart(InspectorScreen.id, analytics_constants.pageReady);
+    }
+
```

Comment:
@jacob314 because the InspectorController is not provided using package:provider like our other controllers, it is recreated in the inspector screen's `initState` method each time the inspector tab is selected. So for example, opening devtools to the inspector, switching to a different tab, then switching back to the inspector tab would trigger this code twice.

We could do one of two things here:
1) time the inspector tree load time for every time the inspector screen is built. This would include the initial load as well as subsequent loads from tab switches. 
2) time the inspector tree load time for _only_ the first load. To do this, we'd store `firstLoadCompleted` as a top level variable instead of as a field on the controller class. This would make more sense if we expect subsequent loads to be faster than the initial load, as that would skew the results of the true "first load" time.

Reply:
Timing only the first load seems reasonable. It would probably also be a good idea to clean up the tech debt that the inspectorController is not provided using package:provider.

Reply:
sg. Filed https://github.com/flutter/devtools/issues/3950 to track.